src/script/main/app.ts

```diff
@@ -443,7 +443,7 @@ export class App {
       // We load all the users the self user is connected with
       await userRepository.loadUsers(selfUser, connections, conversations, teamMembers);
 
-      if (supportsMLS()) {
+      if (this.core.hasMLSDevice) {
```

Comment:
if we have a MLS device it means that:
- the env supportsMLS (previous check)
- the team has MLS enabled
- the user has a valid mls device

(it a more exhaustive check than the previous `supportsMLS` check)